src/components/displaySettings/displaySettings.template.html

```diff
@@ -6,66 +6,75 @@ <h2 class="sectionTitle">
     <div class="selectContainer languageSection hide">
         <select id="selectLanguage" is="emby-select" label="${LabelDisplayLanguage}">
             <option value="">${Auto}</option>
-            <option value="af">Afrikaans</option>
-            <option value="sq">Albanian</option>
-            <option value="ar">Arabic</option>
-            <option value="be-BY">Belarusian</option>
-            <option value="bn_BD">Bengali (Bangladesh)</option>
-            <option value="bg-BG">Bulgarian</option>
-            <option value="ca">Catalan</option>
-            <option value="zh-HK">Chinese (Hong Kong)</option>
-            <option value="zh-CN">Chinese (Simplified)</option>
-            <option value="zh-TW">Chinese (Traditional)</option>
-            <option value="hr">Croatian</option>
-            <option value="cs">Czech</option>
-            <option value="da">Danish</option>
-            <option value="nl">Dutch</option>
+            <option value="af">Afrikaans (af)</option>
+            <option value="ar">العربية (ar)</option>
+            <option value="be-BY">Беларуская (be)</option>
+            <option value="bg-BG">Български (bg)</option>
+            <option value="bn_BD">বাংলা (bn-BD)</option>
+            <option value="ca">Català (ca)</option>
```

Comment:
I like native naming, but I am not sure about adding language codes.

Reply:
Hi Dmitry,
Language codes in case, due to probablity with some problems with the fonts (for example, a user on Reddit recently complained that some of the texts in Malayalam script aren't visible) the corresponding name isn't readable. From here বাংলা (or empty squares) will you understand that this is Bengali (Bangladesh) if you are not Bengali or have studied Bengali, or at least not familiar with the zoo of Indian scripts? Thanks.

Reply:
> Language codes in case, due to probablity with some problems with the fonts (for example, a user on Reddit recently complained that some of the texts in Malayalam script aren't visible) the corresponding name isn't readable.

If user has a problem with fonts, selecting such "bad" language may make the UI completely unreadable/empty.
Need to fix the fonts first, imo.

> From here বাংলা (or empty squares) will you understand that this is Bengali (Bangladesh) if you are not Bengali or have studied Bengali, or at least not familiar with the zoo of Indian scripts?

I suppose users will look for language they understand.
Or are you talking about testing different languages?

Reply:
Yes, mostly last case.

Reply:
_Discussed on Matrix_
I think most of the team (those who answered) are for removing codes: `If you don't recognize the language from the native name, then the language isn't for you...`

Testing different language also requires knowledge of it. Even if you have to test a language you don't understand, you can look the code in the inspector - as long as it happens rarely.

It is also possible to make the names translatable, but you may run into the problem reverting the language back after testing the unknown one.

Reply:
`the team (those who answered) are for removing codes`
OK. No problemo. I will remove the codes.

`It is also possible to make the names translatable, but you may run into the problem reverting the language back after testing the unknown one.`
Agree. And no less important, or perhaps more important, the size of the code, and along with it, the size of files, for example, the Emby.Server.Implementations.dll file, predictably grows in a geometric progression (N-1)^2, where N is the number of languages.